src/cloudai/_core/test_scenario.py

```
@@ -20,7 +20,7 @@
 import itertools
 from dataclasses import dataclass, field
 from pathlib import Path
-from typing import TYPE_CHECKING, Any, List, Optional, Set, Type, Union
+from typing import TYPE_CHECKING, Any, List, Optional, Set, Type, TypeAlias, Union
```

Ruff (0.15.9)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)